monai/losses/image_dissimilarity.py

```diff
@@ -59,9 +59,11 @@ class LocalNormalizedCrossCorrelationLoss(_Loss):
         DeepReg (https://github.com/DeepRegNet/DeepReg)
     """
 
+    @deprecated_arg(name="ndim", since="0.6", msg_suffix="Please use `spatial_dims` instead.")
```

Comment:
The `since` should be `0.6` or `0.7`?

Reply:
It should be 0.6 otherwise the weekly previews before 0.7 doesn't have this warning message